### PR TITLE
Express 102 lesson: add additional resource link

### PR DESCRIPTION
**Pull request by tarwat-uddin**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [x] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
 - [x] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

Added a resource link to a video that helps to give you an brief overview of the application of Mongoose in the Node.js environment.

### CODE REVIEW

File: nodeJS/express-basics/Express-Lesson-2.md
Comment (fortypercenttitanium) [inline nodeJS/express-basics/Express-Lesson-2.md:73]: I'm not sure I'd call a 33 minute video "quick"

```suggestion
- This [crash course video](https://www.youtube.com/watch?v=DZBGEVgL2eE) from Web Dev Simplified gives a run-down of how you should expect to use MongoDB (Mongoose) in Node.js, as well as some advanced things you can do with object schemas.
```

**Review by fortypercenttitanium — CHANGES_REQUESTED** [review]

File: nodeJS/express-basics/Express-Lesson-2.md
Comment (tarwat-uddin) [reply nodeJS/express-basics/Express-Lesson-2.md]: The first 20 minutes of the video goes through the basics of using Mongoose, and the rest is dedicated to the advanced stuff, so I'd reckon you could just skip to parts of the video that you needed to review/learn as a reference - that's was my intention of using the word "quick" here. Thanks for your suggestion btw 🙂

**Review by fortypercenttitanium — APPROVED** [review]

**Review by tarwat-uddin — COMMENTED** [review]
Link seems ready to be merged

**Comment by tarwat-uddin** [comment]
This is my first time submitting a pull request and I'm not sure what's going on, but can someone tell me if I need to do something before this pull request is ready to be merged, or is there nothing to do?  Sorry for my lack of understanding of how pull requests work since I'm not really familiar with this process. Thanks in advance

**Comment by DigitalKnight0** [comment]
@tarwat-uddin There's nothing for you to do now. Don't close this PR again like you did before. Leave it as is, a team member will merge it or let you know if there are changes needed

**Review by fortypercenttitanium — APPROVED** [review]

**Pull request merged**
